lib/lib.dbconfig.php

```diff
@@ -68,6 +68,7 @@ function dbconfig_init()
         $LIBDBCONFIG[$key] = array('value' => $val,
                                    'type' => $row['type'],
                                    'public' => $row['public'],
+                                   'category' => $row['category'],
```

Comment:
This should also be done in the `DOMJudgeBundle\Entity\Configuration` class, but you will probably notice this when you move this page to Symfony 😄 .

Reply:
Done.